.circleci/config.yml

```diff
@@ -44,9 +61,9 @@ workflows:
                 - cimg-aws
                 - cimg-azure
                 - cimg-gcp
-          context: cimg-publishing
+          context: cpe-image-bot-github-creds
```

Comment:
There is no way this works. The CI as-is is trying to use a github token to call the CCI API... 

I'm just going to maintain that behaviour...